### DISCUSSION TITLE
Urgent: Follow-up on High-Severity Vulnerabilities Reported

### DISCUSSION THREAD

**Discussion by dkonis** [open-webui/open-webui#6016] [General]
Hello, [amit-laish](https://github.com/amit-laish) and I reported two high-severity vulnerabilities over a month ago. Please review the submitted vulnerabilities and contact us so we can remediate them.

**Comment by justinh-rahb** [comment]
Who are you talking to? Read our policy on security vulnerability reporting: https://github.com/open-webui/open-webui/blob/main/docs/SECURITY.md#reporting-a-vulnerability

If you're being ignored, you probably didn't follow policy.

**Reply by dkonis** [reply]
Hello justinh-rahb, we followed the instructions and provided a detailed explanation along with a working PoCs, recommendations, and possible fixes. If anything is missing, please contact us instead of ignoring the reports.

Here are the submitted advisories:
https://github.com/open-webui/open-webui/security/advisories/GHSA-4v7r-f4w8-8972
https://github.com/open-webui/open-webui/security/advisories/GHSA-h36q-wpqf-g64q
